### DISCUSSION TITLE
Error response for missing DPoP header

### DISCUSSION THREAD

**Discussion by ashwin-yo** [ietf-tools/datatracker#5146] [Help / Questions]
For the [OAuth DPoP](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop), what should the error response for the missing DPoP header be?

For example:
```
GET /protected-resource HTTP/1.1
Host: resource.example.org
Authorization: DPoP VALID_TOKEN
```

Should the error code be 400 or 401, and what should be the error and error_description here?

**Comment by Rhadamez22** [comment]
Lol600000.89$

**Comment by rjsparks** [comment]
The datatracker repository is not where to report issues with or ask questions about the content of drafts - take those to the working group list (oauth in this case).

**Reply by REDMEE1** [reply]
[!]()
